pbclient/__init__.py

```diff
@@ -36,12 +38,15 @@ def _pybossa_req(method, domain, id=None, payload=None, params=None):
     if method == 'get':
         r = requests.get(url, params=params)
     elif method == 'post':
-        r = requests.post(url, params=params, headers=headers, data=json.dumps(payload))
+        r = requests.post(url, params=params, headers=headers,
+                          data=json.dumps(payload))
     elif method == 'put':
-        r = requests.put(url, params=params, headers=headers, data=json.dumps(payload))
+        r = requests.put(url, params=params, headers=headers,
+                         data=json.dumps(payload))
     elif method == 'delete':
-        r = requests.delete(url, params=params, headers=headers, data=json.dumps(payload))
-    #print r.status_code, r.status_code / 100
+        r = requests.delete(url, params=params, headers=headers,
+                            data=json.dumps(payload))
+    # print r.status_code, r.status_code / 100
     if r.status_code / 100 == 2:
         if r.text and r.text != '""':
```

Comment:
We have one condition that, if holds, then we are returning exactly the same that if it is not (if r.text and r.text != "")